files/view.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import os
+from PyQt4 import QtGui
+from PyQt4 import QtWebKit
+from PyQt4 import QtCore
+from . import io_geojson
+from . import twitter
+import folium
+import random
+
+map_osm = folium.Map(location=[33.4484, -112.0178])
+
+class View(QtGui.QMainWindow):
+
+    def __init__(self):
+        super(View, self).__init__()
+        self.map = None
+        self.web_view = None
+        map_osm.save('/tmp/map.html')
+        self.init_ui
+
+    def init_ui(self):
+        #central widget
+        self.web_view = QtWebkit.QWebView()
+        self.map = folium.Map(location=[33.4484, -112.0178]
+        self.map.zoom_start = 10
+        self.map.save(map_osm.save('/tmp/map.html'))
+        self.web_view.load
+        self.setCentralWidget(self.web_view)
+
+        #Open actions
+        open_action = QtGui.QAction(QtGui.QIcon('Open_Folder.ico'), 'Open', self)
+        open_action.setShortcut('Ctrl+O')
+        open_action.triggered.connect(self.open)
+        
+        #Exit actions
+        exit_action = QtGui.QAction(QtGui.QIcon('exit.png'), 'Exit', self)
+        exit_action.setShortcut('Ctrl+Q')
+        exit_action.triggered.connect(self.close)
+
+        self.statusBar()
+
+        toolbar = self.addToolBar('Exit')
+        toolbar.addAction(exit_action)
+
+        self.setGeometry(300, 300, 700, 700)
+        self.setWindowTitle('Title Window')
+        self.show()
+
```

Comment:
Here is where you would add an open method (or wire line 34 to the open_tweets function on line 50.)  After you open the tweets, you could all the show_tweets methods.  The pieces are all here (show_tweets would need added logic for lat/lon) just a connecting issue.